source/Ship.cpp

```diff
@@ -3550,10 +3550,13 @@ int Ship::TakeDamage(vector<Visual> &visuals, const DamageDealt &damage, const G
 	else if(heat < .9 * MaximumHeat())
 		isOverheated = false;
 
+	double scale = Energy() * 220.;
+	double jamChance = ionization > .1 ? min(0.5, scale ? ionization / scale : 1.) : 0.;
 	// If this ship did not consider itself an enemy of the ship that hit it,
 	// it is now "provoked" against that government.
 	if(sourceGovernment && !sourceGovernment->IsEnemy(government)
-			&& (Shields() < .9 || Hull() < .9 || !personality.IsForbearing())
+			&& (!personality.IsForbearing() || Shields() < .9 || Hull() < .9
+				|| isOverheated || jamChance > 0.1 || slowness > 10.)
```

Comment:
Could this not cause a hostile but forbearing ship to attack you because an environmental hazard affected it?

Reply:
If you attack it whilst its under assault by the hazard, yes
I'll check for the last weapon doing ionic damage to it as well

Reply:
Changed it and did the same for slowness (didnt change overheating though caus if you attack an overheated ship it should be hostile 100%)
If you hit it with ion and its already heavily ion jammed it should react for sure.